scripts/prepare_env.sh

```diff
@@ -43,8 +43,60 @@ function install_dependencies() {
   curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/"${K8S_VERSION}"/bin/linux/amd64/kubectl && \
   chmod +x kubectl && sudo mv kubectl /usr/local/bin/ && sudo ln /usr/local/bin/kubectl /usr/bin/kubectl || true
 
-  curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && \
-  chmod +x minikube && sudo mv minikube /usr/local/bin/ && sudo ln /usr/local/bin/minikube /usr/bin/minikube || true
+  if [[ ! -z ${IS_WSL} && ! -z ${WSL_DISTRO_NAME} ]]; then
+      curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && \
+      chmod +x minikube && sudo mv minikube /usr/local/bin/ && sudo ln /usr/local/bin/minikube /usr/bin/minikube || true
+  else
+      curl -Lo kind https://kind.sigs.k8s.io/dl/v0.9.0/kind-linux-amd64 && \
+      chmod +x kind && sudo mv kind /usr/local/bin/ && sudo ln /usr/local/bin/kind /usr/bin/kind || true
+  fi
+}
+
+function start_docker() {
+  # start docker daemon if docker not running.
+  if ! sudo docker info >/dev/null 2>&1; then
+    if [[ ! -z ${IS_WSL} && ! -z ${WSL_DISTRO_NAME} ]]; then
+      sudo systemctl start docker
+    else
+      sudo dockerd > /dev/null& || true
+    fi
+  fi
+}
+
+function launch_k8s_cluster() {
+  if [[ ! -z ${IS_WSL} && ! -z ${WSL_DISTRO_NAME} ]]; then
```

Comment:
I think you need `"${IS_WSL}"` rather than `{IS_WSL}`. As well for `WSL_DISTRO_NAME`.

Otherwise LGTM.